Subscribers/DoctrineEncryptSubscriber.php

```diff
@@ -210,12 +207,7 @@ public function processFields($entity, $isEncryptOperation = true)
             // Check which operation to be used
             $encryptorMethod = $isEncryptOperation ? 'encrypt' : 'decrypt';
 
-            // Get the real class, we don't want to use the proxy classes
-            if (strstr(get_class($entity), 'Proxies')) {
-                $realClass = ClassUtils::getClass($entity);
-            } else {
-                $realClass = get_class($entity);
-            }
+            $realClass = ClassUtils::getClass($entity);
```

Comment:
doctrine ClassUtils work safely on non-proxy classes. this removes a dependency on implementation details of doctrine.